spec/factories/categories.rb

```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :category do
     name { Faker::Lorem.word }
-    amount { Faker::Number.digit.to_i }
+    amount { Faker::Number.digit.to_f }
```

Comment:
@sinpai  `Faker::Number.digit` It is integer. May be better used `Faker::Number.decimal(2, 2)` or something like this